internal/service/mssql/mssql.go

```diff
@@ -528,7 +529,7 @@ func waitMssqlDeletion(ctx context.Context, config *conn.ProviderConfig, id stri
 		Target:  []string{"deleted"},
 		Refresh: func() (interface{}, string, error) {
 			instance, err := GetMssqlInstance(ctx, config, id)
-
+			// MSSQL deleted, it will respond with a 400 error with a 5001269 return code.
 			if err != nil && !strings.Contains(err.Error(), `"returnCode": "5001269"`) {
```

Comment:
(trivial) The duplicated deleted return code(5001269) condition has been removed from other resources but seems to be left here.

Reply:
5001269 return code error occurs only in MSSQL. It does not occur in other cloudDB.

- 5001269 occurs when a mssql resource is deleted in Terraform
```
400 Bad Request, Body: {
   "responseError": {
     "returnCode": "5001269",
     "returnMessage": "It's a cluster deleted."
   }
}
```
- 5001017 occurs when there is no MSSQL query result. It is assumed that these return codes generate errors after a period of time after deletion.
```
400 Bad Request, Body: {
   "responseError": {
     "returnCode": "5001017",
     "returnMessage": "Unable to lookup cluster instance information."
   }
}
```

Reply:
What I meant was that if we check for error codes in GetMssqlInstance(), don't we need to check them here like we do for other resources?

Reply:
Aha, that's right. Considering that it is also deleted from the console first, it makes sense to check it inside GetMssqlInstance().